conans/client/tools/files.py

```diff
@@ -67,14 +68,15 @@ def unzip(filename, destination=".", keep_permissions=False, pattern=None, outpu
     :param pattern: Extract only paths matching the pattern. This should be a
     Unix shell-style wildcard, see fnmatch documentation for more details.
     :param output: output
+    :param flat: If all the contents are in a single dir, flat that directory.
```

Comment:
to clarify, it always strips only one level?

Reply:
Yes